clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -407,8 +407,6 @@ public void synchronizeKeyToPassive(PassiveSynchronizationChannel<EhcacheEntityM
 
   @Override
   public void createNew() {
-    management.init();
```

Comment:
But so a passive will do the wiring during `createNew` and the call on the registered listener will be a no-op because of the boolean flag?
If you were to clean up the passive entity version as well and always rely on the event, you do not need the flag. Correct?